src/extension.ts

```diff
@@ -291,25 +292,34 @@ export function activate(context: vscode.ExtensionContext) {
 
         let editor = vscode.window.activeTextEditor;
         let glbPath = editor.document.uri.fsPath.replace('.gltf', '.glb');
-        let options = {
-            defaultUri: Uri.file(glbPath),
-            filters: {
-                'Binary glTF': ['glb'],
-                'All files': ['*']
-            }
-        };
-        vscode.window.showSaveDialog(options).then(uri => {
-            if (uri !== undefined) {
-                try {
-                    GlbExport.save(gltf, editor.document.uri.fsPath, uri.fsPath);
-                    vscode.window.showInformationMessage('Glb exported as: ' + uri.fsPath);
-                } catch (ex) {
-                    vscode.window.showErrorMessage(ex.toString());
+        if (fs.existsSync(glbPath)) {
+            const options: vscode.SaveDialogOptions = {
```

Comment:
Do we need this change?  I imagine not every user will want a GLB file saved to the default filename in the same folder as the glTF file.  It makes for inconsistent UX if the app sometimes asks for a save location and sometimes not, depending on the presence of files that may not be in the user's immediate field of view.  I'd rather have the "Save As..." dialog always appear, I hope that's not too much of an inconvenience for your team.

Reply:
I've added a configuration option as you suggested below.